TL improvement + correcting  typos (#436)
07th-mod/higurashi-console-arcs: Update/omot_026.txt

				NULL, "\"...And the end result is this destruction.", Line_WaitForInput);
		ModPlayVoiceLS(4, 5, "ps3/s14/05/vt4bc_rika1291", 256, TRUE);
		OutputLine(NULL, "…因果だな」",

> 				NULL, " ...What terrible karma.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " ...What terrible luck.\"", Line_Normal);
```

	//r…さっき境内で、信号弾を目の前で上げられた時などは、その身体を肉片になるまで引き裂いてやりたいと思えるほどに真っ黒な殺意を抱いたというのに。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "…さっき境内で、信号弾を目の前で上げられた時などは、その身体を肉片になるまで引き裂いてやりたいと思えるほどに真っ黒な殺意を抱いたというのに。",
				NULL, "...When she fired that signal flare right in front of me in the shrine grounds, I was so full of dark bloodlust that I wanted to tear her body apart like a piece of meat.", Line_Normal);
		ClearMessage();

	//rあれほどまでに憎くて、おぞましく思えてならなかったはずが…今になって、どういう心の変化だろう。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "あれほどまでに憎くて、おぞましく思えてならなかったはずが…今になって、どういう心の変化だろう。",

> 				NULL, "Although I must've thought she was so horrendously hateful... what could change my mind that this point?", Line_Normal);

Reviewer: ```suggestion
				NULL, "Although I couldn't help but feel horrendously hateful... what could change my mind that this point?", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#8676cf>羽入</color>", NULL, "<color=#8676cf>Hanyuu</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 12, "ps2/12/221100164", 540, TRUE);
		OutputLine(NULL, "「それを言えば、梨花。",
				NULL, "\"If I told you that, Rika...", Line_WaitForInput);
		ModPlayVoiceLS(4, 12, "ps2/12/221100165", 540, TRUE);
		OutputLine(NULL, "……あなたは僕を頼るだけで、自分で動いたり考えることもなかった。",
				NULL, " you always relied on me, and never thought or acted on your own.", Line_WaitForInput);
		ModPlayVoiceLS(4, 12, "ps2/12/221100166", 540, TRUE);
		OutputLine(NULL, "そして今度こそ、終わってしまっていたのです。",

> 				NULL, " So if I told that this time, it'd all be over.", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, " So if I told you that this time, it'd all be over.", Line_WaitForInput);
```

		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "今度はそれが妙に疎ましくなって、私は羽入に何か言ったら、と苛立たしげに話しかけたとき、——彼女はぽつり、と言った。",

> 				NULL, "So this time, that's what I found strangely alienating. I was irritated and talked to her, like \"say something!\" ...And then, Hanyuu suddenly spoke up.", Line_Normal);

Reviewer: ```suggestion
				NULL, "So this time, that's what I found strangely alienating. I was about to voice my irritation... but then, Hanyuu suddenly spoke up.", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "すると、それを見透かすように羽入はことあるごとに『無駄だ』『決まっていること』と私の黒い心を言葉にするようになり……、",

> 				NULL, "And then, as if Hanyuu could see through that, she'd say things like \"it's futile\" and \"it's already decided,\" putting my black heart into words.", Line_Normal);

Reviewer: ```suggestion
				NULL, "And then, as if Hanyuu could see through that, she'd say things like \"it's useless\" and \"it's already decided,\" putting my black heart into words.", Line_Normal);
```

				NULL, "\"...As time went on, you began to take on a jaded outlook.", Line_WaitForInput);
		ModPlayVoiceLS(4, 12, "ps2/12/221100177", 540, TRUE);
		OutputLine(NULL, "頑張れと言い過ぎると、すぐに止めてしまうようになりました」",

> 				NULL, " When I told you to keep going strong too often, you got to immediately stop trying hard.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Whenever I told you to keep going strong too often, you always immediately stopped trying hard.\"", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "……やっぱり、羽入は私の信じる羽入だった。離れていても、姿を見せなくても、私の幸せのためにこの子は力を振り絞って頑張ってくれていたんだ……！",

> 				NULL, "...I knew it. Hanyuu was just as I believed in. Even though we were apart, even though I couldn't see her, she was still working as hard as she could to secure my happiness...!", Line_Normal);

Reviewer: ```suggestion
				NULL, "...I knew it. Hanyuu was the same Hanyuu I believed in. Even though we were apart, even though I couldn't see her, she was still working as hard as she could to secure my happiness...!", Line_Normal);
```

				NULL, "\"Hee hee... I was surprised at first.", Line_WaitForInput);
		ModPlayVoiceLS(4, 5, "ps3/s14/05/vt4bc_rika1300", 256, TRUE);
		OutputLine(NULL, "でも、あなたの気配をすぐに感じ取ることができたから、わりと早く冷静に返ることができたけどね」",

> 				NULL, " But I sensed your presence soon afterwards, so I was able to calm down relatively quickly.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " But I was able to calm down pretty quickly once I managed to sense your presence.\"", Line_Normal);
```

	//rそして、部活メンバーたちの困難に対しても、常に後手に回って対応が遅れてしまったこと…。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "そして、部活メンバーたちの困難に対しても、常に後手に回って対応が遅れてしまったこと…。",

> 				NULL, "And, even while working to solve the club members' problems, I always took a reactive approach to them and was slow to deal with...", Line_Normal);

Reviewer: ```suggestion
				NULL, "And, even while working to solve the club members' problems, I always kept to a reactive approach, and was slow to deal with them...", Line_Normal);
```

	//rもっとも、そばに羽入の姿が見えなかったことで…今までのようにやり直しがきかないのではと不安になり、気持ちが後ろ向きになってしまったのは確かだが…。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "もっとも、そばに羽入の姿が見えなかったことで…今までのようにやり直しがきかないのではと不安になり、気持ちが後ろ向きになってしまったのは確かだが…。",

> 				NULL, "But, I didn't have Hanyuu by my side... So it's true I was worried that I wouldn't be able to repeat worlds like before, and that thought twisted my emotions all around...", Line_Normal);

Reviewer: ```suggestion
				NULL, "But, I didn't have Hanyuu by my side... So I was definitely worried that I wouldn't be able to repeat worlds like before, and that thought twisted my emotions all around...", Line_Normal);
```

	//r梨花とともに、仲間たちのいる幸せな未来へと進みたい。そして今度こそ、自分は傍観者を決め込まずにみんなと一緒に希望を信じて、戦おう。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "梨花とともに、仲間たちのいる幸せな未来へと進みたい。そして今度こそ、自分は傍観者を決め込まずにみんなと一緒に希望を信じて、戦おう。",

> 				NULL, "I want to move on to a happy future our friends live in, with Rika. And this time, I won't be sitting by as a bystander. I'll believe alongside everyone else, and I'll fight.", Line_Normal);

Reviewer: ```suggestion
				NULL, "I want to move on to a future where our friends can live happily, with Rika. And this time, I won't be sitting by as a bystander. I'll believe alongside everyone else, and I'll fight.", Line_Normal);
```

	//r言霊とでもいうべきなのだろうか、…口に出しているうち、私自身がその疑心暗鬼にとらわれるようになった。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "言霊とでもいうべきなのだろうか、…口に出しているうち、私自身がその疑心暗鬼にとらわれるようになった。",

> 				NULL, "I guess you could call this a supernatural power words have... The more I said those words to her, the more obsessed with that doubt I became.", Line_Normal);

Reviewer: ```suggestion
				NULL, "I guess you could call this a supernatural power that words have... The more I said those words to her, the more obsessed with that doubt I became.", Line_Normal);
```

	//r自分でも信じていないことを教え諭すのは、無意味どころか偽善的な行為だ。だから私たち二人は、無限に続く時の繰り返しをただ漫然と過ごすようになってしまったのだ…。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "自分でも信じていないことを教え諭すのは、無意味どころか偽善的な行為だ。だから私たち二人は、無限に続く時の繰り返しをただ漫然と過ごすようになってしまったのだ…。",

> 				NULL, "Teaching her something I didn't believe in myself wouldn't just be meaningless, it would be hypocritical. That's why the two of us just desultorily continued this repeating cycle...", Line_Normal);

Reviewer: ```suggestion
				NULL, "Teaching her something I didn't believe in myself wouldn't just be meaningless, it would be hypocritical. That's why the two of us just aimlessly continued this repeating cycle...", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 5, "ps3/s14/05/vt4bc_rika1359", 256, TRUE);
		OutputLine(NULL, "「きっと、神様のきつい戒めだったのかもね…だからこうやって、自分の身体と心を引きはがされてしまった」",

> 				NULL, "\"Maybe this was a heavy punishment by God... that my mind and body be separated like this.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"Maybe this was a harsh divine punishment... that my mind and body be separated like this.\"", Line_Normal);
```
